app/controllers/unattended_controller.rb

```diff
@@ -249,7 +249,7 @@ def safe_render(template)
     end
 
     begin
-      render :plain => unattended_render(@unsafe_template_content, @template_name)
+      render :inline => "<%= unattended_render(@unsafe_template_content, @template_name).html_safe %>"
```

Comment:
Thanks, this works for me on HTTPs, not on HTTP though. Does it work for you?

Reply:
What do you mean by that? I don't have HTTPS enabled on my dev setup. Do you mean HTTPS in the foreman_url itself?

Reply:
Nope, my foreman_url is HTTP. I can get the ks template just fine accessing the token URL on HTTPS, but on HTTP (which is really what I need because.. pxe) it doesn't work for me